docs/organizations/managing-security-and-risk.md

```diff
@@ -32,38 +32,91 @@ Each of these areas displays a count of matching items and a **Review** button t
 
 The Security and Risk Management item list displays a list of items, filtered by status and sorted by due date ascending.
 
-To access the item list, access the [dashboard](#dashboard) and click the **Review** button under the area of interest, based on the desired filtering.
+To access the item list, access the [dashboard](#dashboard) and click the **Review** button under an area of interest, based on the desired filtering.
 
 When viewing the item list:
 
--   To update the filtering criteria, click the **Status** drop-down above the list.
+-   To update the filtering criteria, click the **Status** or **Priority** drop-downs above the list.
 -   To find out more about an item, click its **Details** column. This redirects you to a filtered view of the **Repository issues** page, showing only the item of interest.
 
 ![Security and Risk Management item list](images/security-risk-management-item-list.png)
 
+## Managing integrations
+
+Integrations let you detect security vulnerabilities across a number of platforms and display them all as security items on the unified item list.
+
+The following section explains how to integrate with each of the supported platforms.
+
+=== "Codacy"
+
+    Security and Risk Management always displays items caused by Codacy security issues. This feature can't be deactivated.
+
+=== "Jira Cloud"
+
+    To install the Jira Cloud integration:
+
+    1.  Open your organization **Security and Risk** page and click the **Configure** button. This opens the configuration page.
+
+    1.  On the right-hand side of the page, under the Integrations list, find the Jira entry and click the **Install** button. This redirects you to Atlassian's website.
+
+        ![Security and Risk Management Jira integration](images/security-risk-management-integration-jira.png)
+
+    1.  On Atlassian's website, authorize Codacy to access your Atlassian account. Once successful, you're redirected back to Codacy.
+
+        !!! note
+            Use a Jira account with admin permissions when authorizing Codacy. This lets Codacy access all issues.
```

Comment:
Good point. I'm expanding on the reason behind this request, which can also address any user concerns about granting admin permissions (and come up with alternatives, such as creating a non-admin account but with access to all projects, I guess).

```suggestion
            Use a Jira account with admin permissions when authorizing Codacy. This lets Codacy access all issues, since the integration inherits the permissions of the authorizing account.
```